api/src/main/resources/api/service_openapi.yaml

```diff
@@ -20,6 +20,57 @@ paths:
         500:
           description: Service is broken
 
+  '/underlays':
+    get:
+      # TODO authorization
+      security: []
+      summary: Lists the underlays.
+      operationId: listUnderlays
+      tags: [ Underlays ]
+      responses:
+        200:
+          $ref: '#/components/responses/ListUnderlaysResponse'
+
+  '/underlays/{underlay}':
```

Comment:
nit: usually I would call this parameter either an `underlayId` or `underlayName`